packages/openfeature-server-provider/src/factory.ts

```diff
@@ -9,18 +9,13 @@ type ConfidenceProviderFactoryOptions = {
   fetchImplementation: typeof fetch;
   clientSecret: string;
   baseUrl?: string;
-  apply?: {
-    timeout: number;
-  };
 };
 
 export function createConfidenceServerProvider(options: ConfidenceProviderFactoryOptions): Provider {
   const confidenceClient = new ConfidenceClient({
     ...options,
-    apply: !options.apply,
```

Comment:
Could it be the case that users don't want apply at all? For instance if they do their own analysis and don't want to pay for the extra resources used by apply? No need to include in this PR though.

Reply:
interesting. potentially yes. Maybe something we can tackle in a future PR :)